Classes/Repository/RepositoryViewController.swift

```diff
@@ -60,6 +71,53 @@ PrimaryViewController {
         super.viewWillLayoutSubviews()
         feed.viewWillLayoutSubviews(view: view)
     }
+    
+    // MARK: Private API
+    
+    var repoUrl: URL {
+        return URL(string: "https://github.com/\(repo.owner)/\(repo.name)")!
+    }
+    
+    var repoOwnerUrl: URL {
+        return URL(string: "https://github.com/\(repo.owner)")!
```

Comment:
It should be fine for this but moving forward we should prevent referencing this directly in case of service/bot accounts which have a 404 here!